### DISCUSSION TITLE
Pick a Provider to use from HA Automation

### DISCUSSION THREAD

**Discussion by kelvinbrotherston** [music-assistant/support#3028] [Q&A] [answered]
I couldn't see a question or this so...

Is there a way to specify which provider to play from? 
I have 2 YouTube Music accounts for 2 people, but regardless of which account the song or playlist originates from it plays from the provider that was added first. If I disable the 1st provider nothing can play as it sees that disabled, if I delete the 1st one the other is used.

Sooo, is there a way to say in automation that when okay Playlist X it should use Provider X and when playing Playlist Y it should use Provider Y?

**Comment by OzGav** [comment, marked as answer]
No that is not possible. Indeed if you have multiple providers MA will choose the highest quality from all available.

**Reply by kelvinbrotherston** [reply]
That's a shame. My daughter has awful taste and really don't want her poor choices killing my YouTube music recommendations 🤣

**Reply by OzGav** [reply]
More discussion about this is here https://github.com/orgs/music-assistant/discussions/435
